shell/common/persistent_cache.cc

```diff
@@ -289,7 +300,7 @@ static void PersistentCacheStore(fml::RefPtr<fml::TaskRunner> worker,
                                   file_name.c_str(),  //
                                   *mapping)           //
         ) {
-          FML_DLOG(WARNING)
+          FML_LOG(ERROR)
```

Comment:
When this is reached, we already passed `read_only == false` and `IsValid()` check. So a failure is unexpected and we should print an error. The previous `FML_DLOG` will be optimized out in all builds (including debug build) that are shipped with the Flutter framework.